### PR TITLE
Add colombian data example

### PR DESCRIPTION
**Pull request by aladinor**
- [x] Changes are documented in `history.md`

### CODE REVIEW

File: examples/notebooks/Read-plot-Sigmet-data-from-AWS.ipynb [hunk 1]
```diff
@@ -0,0 +1,409 @@
+{
```
Comment (mgrover1) [inline examples/notebooks/Read-plot-Sigmet-data-from-AWS.ipynb:1]: <p>Can you add a link to the bucket metadata on AWS?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openradar/xradar/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (mgrover1) [inline examples/notebooks/Read-plot-Sigmet-data-from-AWS.ipynb:1]: <p>Can you wrap ds in a display(ds)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openradar/xradar/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

**Review by mgrover1 — CHANGES_REQUESTED** [review]
Very nice!! Just a couple of minor comments.

File: examples/notebooks/Read-plot-Sigmet-data-from-AWS.ipynb (hunk 1 above)
Comment (aladinor) [reply examples/notebooks/Read-plot-Sigmet-data-from-AWS.ipynb:1]: <p>Added the bucket metadata link on AWS. In the first version it was there. Not sure when it got lost</p><p> </p>

**Review by mgrover1 — APPROVED** [review]
This looks great! Thanks @aladinor - I went ahead and fixed those conflicts for ya.

**Pull request merged**